app/javascript/packs/user_cabinet.js

```diff
@@ -0,0 +1,10 @@
+$(document).on("turbolinks:load", function () {
+  $(".tablinks").each(function () {
+    $(this).css('background-color', 'none');
+    $(this).removeClass('text-light');
+    if ($(this).attr("href") == window.location.pathname) {
+      $(this).addClass('text-light');
+      $(this).css('background-color', '#448ab5');
```

Comment:
You cab add this style in sone class in css, and apply with addClass, and the same in line 3

Reply:
done